src/3d/qgs3dsceneexporter.cpp

```diff
@@ -669,14 +755,23 @@ void Qgs3DSceneExporter::save( const QString &sceneName, const QString &sceneFol
 
   QFile file( objFilePath );
   if ( !file.open( QIODevice::WriteOnly | QIODevice::Text | QIODevice::Truncate ) )
+  {
+    QgsDebugMsgLevel( QStringLiteral( "Scene can be exported to '%1'. File access error." ).arg( objFilePath ), 1 );
```

Comment:
This is an error, you can use `QgsDebugError()`
Also, 'scene _cannot_ be exported'...

Reply:
done